integration/token/fungible/tests.go

```diff
@@ -477,6 +490,40 @@ func CheckBalance(network *integration.Infrastructure, id string, wallet string,
 	Expect(err).NotTo(HaveOccurred())
 	expectedQ := token2.NewQuantityFromUInt64(expected)
 	Expect(expectedQ.Cmp(q)).To(BeEquivalentTo(0), "[%s]!=[%s]", expected, q)
+
+	// check holding, it must be equal to the balance
+	// first get the enrollment id
+	eIDBoxed, err := network.Client(id).CallView("GetEnrollmentID", common.JSONMarshall(&views.GetEnrollmentID{
```

Comment:
is there a particular reason why this view name uses camelcase whereas the auditor views are in lower case?

Reply:
good point, there is no rule. Shall we decide one and then submit a PR that enforces that?

Reply:
I would just define the rule we using consistently throughout the token sdk. Other users of FSC will maybe have different opinions / naming conventions. Therefore, enforcing something like this doesn't seem to be desirable - I believe.